src/runtime_src/core/tools/common/xball

```diff
@@ -197,7 +198,7 @@ for device in working_devices:
   print("\n")
   print("=====================================================================")
   print("%d / %d [%s] : %s" % (device_count, len(working_devices), device["bdf"], device["vbnv"]))
-  cmd = os.environ['XILINX_XRT'] + "/bin/" + xrt_app + " --device " + device["bdf"] + " " + prog_args
+  cmd = xrt_bin_dir + "/" + xrt_app + " --device " + device["bdf"] + " " + prog_args
```

Comment:
This seems to be a lot of work and rather kludgy for it no longer depends on the XILINX_XRT environment.

Why can't the user use the following command:
`$ sudo -E xball xbmgmt examine`

Note
I can see this script being updated to detect if XILINX_XRT is valid and if not requesting the user to either set it or use the -E option.